Assets/Scripts/EnemyAI/EnemyHealth.cs

```diff
@@ -16,6 +16,12 @@ public class EnemyHealth : MonoBehaviour, IHealth
 
     private bool hasSpawned;
 
+    //If you are seeing this I forgot to go look for an enemy type enum
+    public string enemyType;

```

Comment:
You said to block on making this an enum so this is me doing that

Reply:
https://github.com/jacobmcleman/LD46/pull/30/commits/053c13cddaed007ba5d55564b142b2383f1bf926